1 6.48e-06 terminate called after throwing an instance of 'std::system_error' what(): Resource temporarily unavailable
100 5.588e-06 0.000364683
10000 6.1844e-05 0.00120293
1000000 0.00586871 0.00093316
100000000 0.585619 0.0641372
